### PR TITLE
Add reference to localnet in project README

### PR DESCRIPTION
**Pull request by travisperson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,9 +176,9 @@ To see a full list of commands, run `go-filecoin --help`.
 
 ### Advanced usage
 
-#### Run Multiple Nodes with IPTB
+#### Setting up a localnet
```
Comment (phritz) [inline README.md]: please also change link on line 32ish

File: README.md [hunk 2]
```diff
@@ -176,9 +176,9 @@ To see a full list of commands, run `go-filecoin --help`.
...
 
-The [`localfilecoin` IPTB plugin](https://github.com/filecoin-project/go-filecoin/tree/master/tools/iptb-plugins) provides an automation layer that makes it easy to run multiple filecoin nodes. For example, it enables you to easily start up 10 mining nodes locally on your machine.
+The localnet FAST binary tool allows users to quickly, and easily setup a local network on the users computer. Please refer to the [localnet README](https://github.com/filecoin-project/go-filecoin/tree/master/tools/fast/bin/localnet#localnet) for more information. The localnet tool is only captabile when built from the same git ref as the targeted `go-filecoin` binary.
```
Comment (frrist) [inline README.md]: nit: captabile -> comptable

File: README.md [hunk 3]
```diff
@@ -176,9 +176,9 @@ To see a full list of commands, run `go-filecoin --help`.
 
 ### Advanced usage
 
-#### Run Multiple Nodes with IPTB
+#### Setting up a localnet
 
-The [`localfilecoin` IPTB plugin](https://github.com/filecoin-project/go-filecoin/tree/master/tools/iptb-plugins) provides an automation layer that makes it easy to run multiple filecoin nodes. For example, it enables you to easily start up 10 mining nodes locally on your machine.
+The localnet FAST binary tool allows users to quickly, and easily setup a local network on the users computer. Please refer to the [localnet README](https://github.com/filecoin-project/go-filecoin/tree/master/tools/fast/bin/localnet#localnet) for more information. The localnet tool is only comptable when built from the same git ref as the targeted `go-filecoin` binary.
```
Comment (mishmosh) [inline README.md]: minor nit: no comma between "quickly and easily"

File: tools/fast/bin/localnet/README.md [hunk 4]
```diff
@@ -14,6 +14,8 @@ localnet -shell
 The localnet tool expects that you can already build `go-filecoin`. Please refer
 to the README in the root of this project for details.
 
+localnet is only comptable with `go-filecoin` binaries built from the same git ref.
```
Comment (mishmosh) [inline tools/fast/bin/localnet/README.md]: typo: "compatible"

**Pull request merged**